### ISSUE TITLE
Tab List Button dropdown menu width

### ISSUE THREAD

**Issue by bastien30** [codefisher/toolbar_buttons#8]
First thanks for all theses very usefull (web)extensions !

Is that possible to add the "maximum width of the button's menu" option (like in "Undo Close Tab Button") in "Tab List Button" ?
It would be awsome ;)

Thanks !

**Comment by codefisher** [comment]
Yes it is.  I want to make a lot of changes to the extension, when I do I should include this among them.

**Comment by bastien30** [comment]
Thanks ! :D
